server/src/services/boards/columns.go

```diff
@@ -51,10 +65,14 @@ func (s *BoardService) ListColumns(ctx context.Context, boardID uuid.UUID) ([]*d
 	return dto.Columns(columns), err
 }
 
-func (s *BoardService) UpdatedColumns(board uuid.UUID, columns []database.Column) {
-	err := s.realtime.BroadcastToBoard(board, realtime.BoardEvent{
+func (s *BoardService) UpdatedColumns(board uuid.UUID) {
+	dbColumns, err := s.database.GetColumns(board)
+	if err != nil {
+		logger.Get().Errorw("unable to retrieve columns in updated notes", "err", err)
+	}
```

Comment:
Shouldn't we also return here, since dbColumns should be nil

Reply:
yes